test/ringpop-assert.js

```diff
@@ -92,6 +92,15 @@ function waitForPing(t, tc) {
     };
 }
 
+function fasterWaitForEmptyPing(t, tc) {
```

Comment:
You should document why you are doing this, as the name of the function is not enough. Specifically:

1. Why are you sending pings to the SUT? I know that this will cause the disseminator to drain, but it's not obvious. You should comment that.
2. Why 30? I guess this has something to do with the p-value, but I am not sure. You should also give a reason for that number here.